### DISCUSSION TITLE
How I can hide folder structure from the link in the quarto blog?

### DISCUSSION THREAD

**Discussion by danieltomasz** [quarto-dev/quarto-cli#6418] [Q&A] [answered]
### Description

I want to have my posts  md sources with  local  folders that reflect the data like `blog/posts/2023-08-03-my-first-post` but I want them to have word facing link without date, like `mysite.com/my-first-post`, how I can achieve that?

I dont want to create a hierarchy of folders, I want to be able to change the local folder structure with destroying the  web facing link links.

**Comment by danieltomasz** [comment, marked as answer]
I will experiment with aliases
https://quarto.org/docs/websites/website-navigation.html#redirects

**Reply by mcanouil** [reply]
That's the way to go unless you are willing to change your project directory structureW
